common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -104,6 +95,11 @@ private static void initializeFeature(Feature feature) {
             ConfigManager.registerHolder(new ConfigHolder(feature.getClass(), f, f.getAnnotation(Config.class)));
         }
 
+        // register overlays
+        for (Field f : FieldUtils.getFieldsWithAnnotation(feature.getClass(), Overlay.class)) {
```

Comment:
So are you supposed to declare Overlay fields in a Feature? As I said: we need an example on how you are planning to use this!